cmd/snap/cmd_login.go

```diff
@@ -91,20 +91,23 @@ func requestLoginWith2faRetry(email, password string) error {
 	}
 }
 
-func (x *cmdLogin) Execute(args []string) error {
-	if len(args) > 0 {
-		return ErrExtraArgs
-	}
-
-	email := x.Positional.Email
+func requestLogin(email string) error {
 	fmt.Fprint(Stdout, i18n.G("Password: "))
-	password, err := terminal.ReadPassword(0)
+	password, err := terminal.ReadPassword(Terminal)
```

Comment:
I'd'a' thunk mocking `terminal.ReadPassword` was a easier and cleaner approach, but this works as well